pkgs/applications/office/skrooge/default.nix

```diff
@@ -24,6 +24,8 @@ mkDerivation rec {
     kactivities karchive kguiaddons knotifyconfig krunner kwindowsystem libofx
   ];
 
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
```

Comment:
This is added automatically by cmake setup-hook. Don't add manually.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L52